### DISCUSSION TITLE
如何添加 --no-sandbox  命令行参数

### DISCUSSION THREAD

**Discussion by xiaoxiaohuayu** [alex8088/electron-vite#309] [Q&A]
How to add -- no sandbox command-line parameters

**Comment by alex8088** [comment]
https://cn.electron-vite.org/guide/dev.html#%E4%BC%A0%E9%80%92-cli-%E5%8F%82%E6%95%B0%E7%BB%99-electron-%E5%BA%94%E7%94%A8%E7%A8%8B%E5%BA%8F

下一个版本将新增加此cli此命令
